tests/infer/test_util.py

```diff
@@ -44,3 +46,22 @@ def test_multi_frame_tensor():
     for name, expected_sum in expected.items():
         actual_sum = actual.sum_to(stacks[name])
         assert_equal(actual_sum, expected_sum, msg=name)
+
+
+@pytest.mark.parametrize('max_particles', [250 * 1000, 500 * 1000])
+@pytest.mark.parametrize('scale_krange', zip([0.50, 0.95], [(0.7, 0.9), (0.05, 0.2)]))
```

Comment:
nit: You can write this directly using pytest:
```py
@pytest.mark.parametrize('scale,krange', [(0.5, (0.7, 0.9)),
                                          (0.95, (0.05, 0.2))])
...
def test_psis_diagnostic(scale, krange, ...):
```

Reply:
done